RFS/src/testFixtures/java/org/opensearch/migrations/bulkload/http/ClusterOperations.java

```diff
@@ -158,7 +158,7 @@ public void createES6LegacyTemplate(final String templateName, final String patt
             "  }\r\n" + //
             "}";
 
-        final var createRepoRequest = new HttpPut(this.clusterUrl + "/_template/" + templateName);
+        final var createRepoRequest = new HttpPut(this.clusterUrl + "/_template/" + templateName + "?include_type_name=true");
```

Comment:
It seems like this will create a template w/ some extra layers in the json.  From there, in the e2e test, we're trying to show that the intermediate type, which was hopefully there, no longer is, is that right?

Putting some assertions on the original template to show that it exhibited the behavior that you're trying to tickle would be helpful.

Reply:
There's no behavior change with `?include_type_name=true` in this operation, we're just accepting that we're intentionally including the type in the put which will not log the warning message for this operation.

Reply:
Code is parameterized, so putting a sanity check to assert that intermediate type mappings are present isn't feasible to do cleanly.